dev-libs/aws-sdk-cpp/aws-sdk-cpp-1.6.23.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="AWS SDK for C++"
+HOMEPAGE="https://aws.amazon.com/sdk-for-cpp/"
+SRC_URI="https://github.com/aws/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+MODULES="access-management acm acm-pca alexaforbusiness \
```

Comment:
I just wanted to thank you a lot for corrections/changes. I really do appreciate the effort you've to put into this.

I've since updated the ebuild to incorporate the changes as recommended. Please let me know if there are any other things I should fix (or if you have a recommendation on how to re-structure this ebuild)